frontend/SplitView.tsx

```diff
@@ -0,0 +1,80 @@
+
+import { Box } from "@hope-ui/solid";
+import { ParentComponent, onMount } from "solid-js";
+
+
+const SplitView: ParentComponent<{
+}> = (props) => {
+    onMount(() => {
+        document.addEventListener('DOMContentLoaded', function () {
```

Comment:
This is a whole bunch of non-reactive code - the DOM is directly manipulated instead of reactive state maintained for various variables. That's okay as it's isolated and only affects a limited area of the DOM but I wanted to highlight it as rather "alien" to the codebase.